.pipelines/templates/template-push-images-to-acr.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  workingDirectory: ""
+  rpImageACR: ""
+steps:
+  - script: |
+      cd ${{ parameters.workingDirectory }}
+
+      export RP_IMAGE_ACR=${{ parameters.rpImageACR }}
+
+      az acr login --name "$RP_IMAGE_ACR"
+      make publish-image-aro
+    displayName: "⚙️ Build and push images to ACR"
```

Comment:
please would you set your editor to add a newline to the end of text files?  otherwise we get last line changes all the time and `cat`ing files on the console is inconvenient.